### PR TITLE
Add cli tests for ACSes

### PR DESCRIPTION
**Pull request by vsedmik**
Adds basic cli tests (CRUD and validations) for Alternate Content Sources.

### CODE REVIEW

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_acs.py

**Comment by vsedmik** [comment]
1) Not sure why does the CI/CQ fail - any clue?
2) The PRT failure probably relates to this change https://github.com/Katello/katello/pull/10485 
which is merged in stream but not in 6.13 (locally all tests passed against 6.13 SAT). I will try to rerun against stream and update accordingly.

**Review by pondrejk — APPROVED** [review]

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_acs.py

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_acs.py

**Comment by vsedmik** [comment]
Updated validation messages which slightly differ between 6.13 and stream (see https://github.com/Katello/katello/pull/10485). PRT passes now.

File: tests/foreman/cli/test_acs.py [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_alphanumeric
+
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo.cli.factory import make_product
+from robottelo.cli.factory import make_repository
```
Comment (ogajduse) [inline tests/foreman/cli/test_acs.py:24]: Use `target_sat.cli_factory.make_product` and `target_sat.cli_factory.make_repository` and remove these imports.

File: tests/foreman/cli/test_acs.py [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+from robottelo.cli.repository import Repository
```
Comment (ogajduse) [inline tests/foreman/cli/test_acs.py:25]: Use `target_sat.cli.Repository` and remove this import.

File: tests/foreman/cli/test_acs.py [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+            }
+        )
+
+    new_acs = target_sat.cli.ACS.create(params)
+
+    # List
+    list = target_sat.cli.ACS.list({'search': f'name={new_acs["name"]}'})
+    assert len(list) == 1
+    assert list[0]['id'] == new_acs['id']
+    assert list[0]['type'] == new_acs['alternate-content-source-type']
```
Comment (ogajduse) [inline tests/foreman/cli/test_acs.py:114]: non-blocking, having an assertion messages present helps better readability and possibly TFA

File: robottelo/constants/repos.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 """Only External Repos url specific constants module"""
 
-
+PULP_FIXTURE_ROOT = 'https://fixtures.pulpproject.org/'
```
Comment (ogajduse) [inline robottelo/constants/repos.py:3]: This is more of your team's decision, but I recommend moving these fixtures to our hosted mirror with content - https://github.com/SatelliteQE/fedorapeople-repos. Not sure how relevant this suggestion is relative to what this test is trying to do, but my concerns are possible network issues and external service availability.

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_acs.py

**Review by sambible — APPROVED** [review]
Nice work.

**Comment by vsedmik** [comment]
rebased

**Review by omkarkhatavkar — APPROVED** [review]

File: tests/foreman/cli/test_acs.py [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+
+:Requirement: AlternateContentSources
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: AlternateContentSources
+
+:Team: Phoenix
```
Comment (devendra104) [inline tests/foreman/cli/test_acs.py:11]: @jyejare @vsedmik Can you correct the team name here, it should be `phoenix-content`

**Pull request merged**